packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -292,6 +239,10 @@ Mocked aad configuration orchestration content. Module path: './teamsFx/aadConfi
         AadParameter: `${TestFileContent.aadParameterValue}`,
       })
     );
+    expect(await fs.pathExists(path.join(projectArmTemplateFolder, TestFilePath.configFileName))).to
```

Comment:
should we expect this to be true? The bot plugin requires config

Reply:
261 line will add bot plugin, current there has no any other plugins except frontend hosting. 
so there is no any config files

Reply:
thanks